### DISCUSSION TITLE
how to set a global varibale in pocl?

### DISCUSSION THREAD

**Discussion by KeyBor** [pocl/pocl#1361] [Q&A]
i just want to add a new global var in pocl driver level, but when i add it, any case will give the result that get platform id failed and return -1001

**Comment by jansol** [comment]
If clGetPlatformIDs fails that often means that your driver library fails to load. You can learn more about what is going on by setting the `POCL_DEBUG=all` or `LD_DEBUG=libs` environment variable.
